File: src/main/java/net/punchtree/loquainteractable/metadata/commands/MetadataWandCommand.java
package net.punchtree.loquainteractable.metadata.commands;

import org.bukkit.command.Command;
import org.bukkit.command.CommandExecutor;
import org.bukkit.command.CommandSender;
import org.bukkit.entity.Player;

import net.punchtree.loquainteractable.metadata.editing.MetadataWand;

public class MetadataWandCommand implements CommandExecutor {

	@Override
	public boolean onCommand(CommandSender sender, Command cmd, String label, String[] args) {
		
		if ( ! (sender instanceof Player)) return true;
		Player player = (Player) sender;
		
		MetadataWand.giveMetadataWandItem(player);
		
		return true;
	}

	
}

File: src/main/java/net/punchtree/loquainteractable/metadata/commands/MetadataWandCommand.kt
package net.punchtree.loquainteractable.metadata.commands

import net.punchtree.loquainteractable.metadata.editing.MetadataWand.giveMetadataWandItem
import org.bukkit.command.Command
import org.bukkit.command.CommandExecutor
import org.bukkit.command.CommandSender
import org.bukkit.entity.Player

object MetadataWandCommand : CommandExecutor {
    override fun onCommand(sender: CommandSender, cmd: Command, label: String, args: Array<out String>): Boolean {
        if (sender !is Player) return false

        giveMetadataWandItem(sender)

        return true
    }
}
